scripts/postgres_init.sql

```diff
@@ -1,11 +1,16 @@
-/* Entrypoint script for postgres container to set up databases and users for 
+/* Entrypoint script for postgres container to set up databases and users for
 docker-compose setup */
 
-CREATE DATABASE metadata_db;
+CREATE DATABASE metadata; -- Used by metdata-service (called "metdata" in cloud-automation)
```

Comment:
let's update this doc and make it clear which one is which there too https://github.com/uc-cdis/compose-services/blob/9d741aacbb1b3c1f9113c2941638ad2fa9019b93/README.md#some-database-info

Reply:
✔️